### ISSUE TITLE
Support Additional Lanterns mod to add extra lantern support.

### ISSUE THREAD

**Issue by v3xed-v0id** [txnimc/ImmersiveLanterns#8]
link to mod: https://modrinth.com/mod/additional-lanterns

would like to see lanterns from other mods be supported, thank you for reading.

**Comment by nthxny** [comment]
Yes, this is planned, I want to add a tag that will allow users to add their own lanterns (with some stuff like this supported by default). The mod doesn't actually care what item is in the slot.

**Comment by v3xed-v0id** [comment]
> Yes, this is planned, I want to add a tag that will allow users to add their own lanterns (with some stuff like this supported by default). The mod doesn't actually care what item is in the slot.

Thank u for replying, i like ur mod a lot and already have been using it as my primary hip lantern mod :)

i hope development goes well, best wishes!

**Comment by TheVillain-Inc** [comment]
Ahoy!

Been using this mod for a few days - love it. Really awesome. 

This mod - 
https://www.curseforge.com/minecraft/mc-mods/curious-lanterns

Does a similar thing and has more compat for other lanterns, if you do end up adding in a 'tag' system I'd be happy to help will generating jsons to support other lanterns.

**Comment by rjshadowface** [comment]
> Yes, this is planned, I want to add a tag that will allow users to add their own lanterns (with some stuff like this supported by default). The mod doesn't actually care what item is in the slot.

Hi,
Is there any progress on the tagging capability for modded lanterns?

**Comment by xinwolfthorn** [comment]
> > Yes, this is planned, I want to add a tag that will allow users to add their own lanterns (with some stuff like this supported by default). The mod doesn't actually care what item is in the slot.
> 
> Hi, Is there any progress on the tagging capability for modded lanterns?

i second this, as this could add so much more character customization options..
